package.json

```diff
@@ -11,15 +11,15 @@
   "dependencies": {
     "@iconscout/unicons": "^4.0.8",
     "@types/node": "20.4.1",
-    "@types/react": "18.2.14",
-    "@types/react-dom": "18.2.6",
+    "@types/react": "^18.2.14",
+    "@types/react-dom": "^18.2.6",
     "autoprefixer": "10.4.14",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.9",
     "next": "13.4.9",
     "postcss": "8.4.25",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
Don't change these

Reply:
This was changed automatically - you are welcome to change it back. It just allows for folks like me to contribute easier as it opens up version to this versioning or higher.